XMonad/Actions/TreeSelect.hs

```diff
@@ -654,6 +654,8 @@ drawStringXMF display window visual colormap gc font col x y text = case font of
 -- | Convert 'Pixel' to 'XRenderColor'
 --
 -- Note that it uses short to represent its components
+--
+-- This fixes a bug with font rendering, caused by the haskell bindings for xorg-xft.
```

Comment:
Oh come on, this doesn't even mention colors! Why don't you just take all those comments of ours and summarize them into a couple sentences. No need to invent new words, change package names, or anything.